polaris_wyre/wyre/wyre.py

```diff
@@ -0,0 +1,35 @@
+from typing import Tuple
+
+from polaris_wyre.wyre.dtos import TransferData
+from .api import TEST_BASE_URL, WyreAPI
+
+COMPLETED_STATUS = "COMPLETED"
+FAILED_STATUS = "FAILED"
+
+
+# TODO: add methods description
+class Wyre:
```

Comment:
Why split into `Wyre` and `WyreAPI` classes instead of making a single class? I ask that because right now this class has very few methods, and I understand that there may be a good reason for it, but it's not obvious right now.

Reply:
We have this to separate the code into some layers. The `WyreAPI` class only handles the request of the API and its response.

The `Wyre` class processes the API response. For example, it finds Stellar's address of the account. It can be a little redundant since it has a few lines of code here, but we gain on organization, maintainability, and testability.

Reply:
Got it. With that in mind, I got a suggestion - how about making the `Wyre` class responsible for calling the `_handle_response` method? That way it would add the possibility of, for example, creating a method in `Wyre` that calls another method for handling the same response. Here's what I mean:

```
class Wyre:

...

def create_transfer(self, transfer_data: TransferData) -> dict:
    response = self.wyre_api.create_transfer(transfer_data)
    return self._handle_response(response)

# A possible future method.
def alternative_create_transfer(self, transfer_data: TransferData) -> dict:
    response = self.wyre_api.create_transfer(transfer_data)
    return self._alternative_handle_response(response)
```

With this change, though, there would be a change in philosophy - the `WyreAPI` class would be the "resquest maker" and `Wyre` would be the "response handler". I think that this would be a valid separation of concerns, but may not be what you guys have in mind. Still, as it is right now, `Wyre` is already a bit of a "response handler", since both `get_account` and `get_stellar_transaction_id` are handling response data, which means that the class has knowledge of the structure of some of the responses' body. Let me know what you think.